posthog/api/organization_integration.py

```diff
@@ -0,0 +1,63 @@
+from rest_framework import mixins, serializers, viewsets
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from posthog.api.routing import TeamAndOrgViewSetMixin
+from posthog.api.shared import UserBasicSerializer
+from posthog.models.organization_integration import OrganizationIntegration
+from posthog.permissions import OrganizationAdminWritePermissions
+
+
+class OrganizationIntegrationSerializer(serializers.ModelSerializer):
+    """Serializer for organization-level integrations."""
+
+    created_by = UserBasicSerializer(read_only=True)
+
+    class Meta:
+        model = OrganizationIntegration
+        fields = [
+            "id",
+            "kind",
+            "integration_id",
+            "config",
+            "created_at",
+            "updated_at",
+            "created_by",
+        ]
+        read_only_fields = ["id", "created_at", "updated_at", "created_by"]
```

Comment:
This means people can edit the kind and integration id. Please fix this and prevent people from updating these fields.